apps/api/src/modules/applications/applications.service.ts

```diff
@@ -35,23 +36,18 @@ export class ApplicationsService extends CoreService<Application> {
     applicationInput: CreateApplicationInput,
     authorizationHeader: Request,
   ): Promise<Application> {
-    const isAdmin = await this.checkAdminUser(authorizationHeader);
+    const userEntry = await this.getUserEntryFromToken(authorizationHeader);
```

Comment:
Don't pass header here.
Get data related to user from `context.req.user`  pring context.req you sould have user or something similar based on the details returned in strategy file.
Get user id from that

Reply:
Updated bbed12eb7057e1f4c0b105a104faf2ccd967530c